x-pack/plugins/global_search_bar/public/components/search_bar.tsx

```diff
@@ -136,7 +136,7 @@ const resultToOption = (
   const { id, title, url, icon, type, meta = {} } = result;
   const { tagIds = [], categoryLabel = '' } = meta as { tagIds: string[]; categoryLabel: string };
   // only displaying icons for applications
-  const useIcon = type === 'application';
+  const useIcon = type === 'application' || type === 'integration';
```

Comment:
Core team, there's no tests on this logic right now so I did not extend them for this very small change.

Reply:
NIT: please, amend the comment `// only displaying icons for applications and Fleet's integrations` 😇